plugins/embed-optimizer/detect.js

```diff
@@ -73,7 +52,7 @@ export async function initialize( { isDebug } ) {
  * @param {FinalizeArgs} args Args.
  */
 export async function finalize( {
-	isDebug,
+	logger,
```

Comment:
See note above passing the logger functions directly without needing the `logger` object.